packages/block-editor/src/components/block-list/insertion-point.js

```diff
@@ -100,8 +96,7 @@ function InsertionPointPopover( {
 				'vertical';
 
 			return {
-				previousElement: getBlockDOMNode( previous, ownerDocument ),
-				nextElement: getBlockDOMNode( next, ownerDocument ),
+				previousClientId: previous,
```

Comment:
Note that previously the element wouldn't update if it changed because it's memoized by client ID.